bears/julia/JuliaLintBear.py

```diff
@@ -6,6 +6,8 @@
 class JuliaLintBear(LocalBear, Lint):
     executable = 'julia'
     arguments = '-e \'import Lint.lintfile; lintfile({filename})\''
+    prerequisite_command = ['julia', '-e', '\'import Lint.lintfile;\'']
+    prerequisite_fail_msg = "Please ensure you have Lint package installed."
```

Comment:
please add a command that I can run to install it, that'd be way better for the suer